pkg/report/report.go

```diff
@@ -82,6 +83,26 @@ type ExecutorInfo struct {
 	ExecID int // The program the syz-executor was executing.
 }
 
+var syzLinuxCommRe = regexp.MustCompile(` Comm: syz\.(\d+)\.(\d+) `)
+
+func (rep *Report) setExecutorInfo() {
+	match := syzLinuxCommRe.FindSubmatch(rep.Report)
```

Comment:
`Report` is assumed to contain the generic report-parsing functionality. Linux-related stuff better stay in `linux.go`.

Reply:
The process name being printed as `Comm: name` is Linux-specific

Reply:
If you don't like it, I'll remove this part.